doc/howto/container_gpu_passthrough_with_docker.md

```diff
@@ -0,0 +1,147 @@
+
+1. Create a `cloud-init` script that installs the Docker runtime, the [NVIDIA Container Toolkit](https://github.com/NVIDIA/nvidia-container-toolkit), and a script to run a test [TensorRT](https://github.com/NVIDIA/TensorRT) workload:
+
+        #cloud-config
+        package_update: true
+        packages:
+          - docker.io
+        write_files:
+          - path: /root/run_tensorrt.sh
+            permissions: "0755"
+            owner: root:root
+            content: |
+            #!/bin/bash
+            echo "OS release,Kernel version"
+            (. /etc/os-release; echo "${PRETTY_NAME}"; uname -r) | paste -s -d,
+            echo
+            nvidia-smi -q
+            echo
+            exec bash -o pipefail -c "
+            cd /workspace/tensorrt/samples
+            make -j4
+            cd /workspace/tensorrt/bin
+            ./sample_onnx_mnist
+            retstatus=\${PIPESTATUS[0]}
+            echo \"Test exited with status code: \${retstatus}\" >&2
+            exit \${retstatus}
+            "
+        runcmd:
+          - systemctl start docker
+          - systemctl enable docker
+          - usermod -aG docker root
+          - curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | sudo gpg
+            --dearmor -o /usr/share/keyrings/nvidia-container-toolkit-keyring.gpg
+          - curl -s -L
+            https://nvidia.github.io/libnvidia-container/stable/deb/nvidia-container-toolkit.list
+            | sed 's#deb https://#deb
+            [signed-by=/usr/share/keyrings/nvidia-container-toolkit-keyring.gpg]
+            https://#g' | tee /etc/apt/sources.list.d/nvidia-container-toolkit.list
+          - sed -i -e '/experimental/ s/^#//g'
+            /etc/apt/sources.list.d/nvidia-container-toolkit.list
+          - apt-get update
+          - DEBIAN_FRONTEND=noninteractive apt-get install -y nvidia-container-toolkit
+          - nvidia-ctk runtime configure --runtime=docker
+            --config=/etc/docker/daemon.json
+          - systemctl restart docker
+
+1. Apply this `cloud-init` setup to your instance:
+
+        lxc config set t1 cloud-init.user-data - < cloud-init.yml
+
+1. Start the instance:
+
+        lxc start t1
+
+1. Wait for the `cloud-init` process to finish:
+
+        lxc exec t1 -- cloud-init status --wait
+
+1. Once cloud-init is finished, open a shell in the instance:
+
+        lxc exec t1 -- /bin/bash
+
+1. Edit the NVIDIA container runtime to avoid using `cgroups`:
+
+        sudo nvidia-ctk config  --in-place --set nvidia-container-cli.no-cgroups
+
+1. If you use an iGPU and that your NVIDIA container runtime is not automatically enabled with CSV mode (needed for NVIDIA Tegra board), you can enable it manually:
+
+        sudo nvidia-ctk config --in-place --set nvidia-container-runtime.mode=csv
+
+1. Now, run the inference workload with Docker:
+
+    - If you setup a dGPU pass-through:
```

Comment:
```suggestion
    - If you set up a dGPU pass-through:
```
(It's "to set up" as a verb, but "a setup" as a noun.)

Reply:
fixed